src/variables.jl

```diff
@@ -11,6 +11,7 @@ struct Variable <: AbstractJuMPScalar
     m::Model
     index::MOIVAR
 end
+coeftype(::Variable) = Float64
```

Comment:
This is a sketchy helper function for the macros. I think it should be defined in `macros.jl`.